landing-pages/site/content/en/blog/experience-with-airflow-as-an-outreachy-intern.md

```diff
@@ -0,0 +1,29 @@
+title: "Journey with Airflow as an Outreachy Intern"
+linkTitle: "Journey with Airflow as an Outreachy Intern"
+author: "Omair Khan"
+github: "OmairK"
+linkedin: "omairkhan64"
+description: ""
+tags: [Community]
+date: "2020-08-15"
+draft: true
+
+[Outreachy](https://www.outreachy.org/) is a program which organises three months paid internships with FOSS
+projects for people who are typically underrepresented in those projects.
+
+The first thing I had to do was choose a project under an organisation. After going through all the projects
+I chose “Extending the REST API of Apache Airflow”, because I had a good idea of REST API(s) so I thought it
+would be easier to get started with the contributions. I had never contributed to FOSS before so this part
+was overwhelming but there were plenty of issues labelled “good first issues” with detailed descriptions and
+some even had code snippets so luckily that nudged me in the direction of my first PR.
+
+My first PR made me familiar with the feedback loop and those feedbacks on my subsequent PRs were the focal
```

Comment:
Consider linking to your PR on "my first PR" - I bet folks would love to see it! Additionally, consider listing your additional ones somewhere - not only will it give users some good examples of healthy PR discussions, it'll be a nice reference for you in the future :)

Reply:
That sounds fantastic I have made this PR WIP for now. I want to add some more lines to this blog, I kinda skimped on some incidents related to my first PR :sweat_smile:.